Orso.Arpa.Application/PersonApplication/PersonDto.cs

```diff
@@ -40,6 +40,9 @@ public class PersonDto : BaseEntityDto
 
         public string Birthplace { get; set; }
 
+        [IncludeForRoles(RoleNames.Staff)]
```

Comment:
Dürfen das Performer nicht sehen? In dem Fall muss auch noch das Attribut `[JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingDefault)]` hinzugefügt werden

Reply:
Richtig, es soll nur für Staff sichtbar sein.. das suffix "Team" hab ich tatsächlich vergessen ;-)

Reply:
Ja, das war ne bewusste Entscheidung: background zu mupro ist tatsächlich auf die eigenschaft des performers am instrument/stimme begrenzt. in der person gehts eher um allgemeinen background (wie kam die person zu orso, wie ist die so drauf, sonstige praktische infos wie bruder/schwester von 2.violine xy etc.). im mupro background steht zeugs wie "mehrere wettbewerbe gewonnen" , "letzte saison aushilfe beim swr" usw.

Reply:
eyy.. schon wieder die DB Migration.. hatte mir eingebildet, daß ich das nur brauche wenn ich Seed-Data was ändere.. aber auch da muss ich ja jetzt eh was rein tun, hole ich nach ,-)

Reply:
Ich nehme an, es genügt wenn ich alle "Background"s rename in"BackgroundTeam" oder muss ich da noch was beachten? @VILLAN3LL3

Reply:
statt "PersonsSeedData" meintest du vermutlich "PersonTestSeedData", kann das sein? @VILLAN3LL3 
im ersteren erscheint mir das unlogisch..

Reply:
Hier fehlt noch das Attribut `[JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingDefault)]`